Dockerfile

```diff
@@ -20,28 +20,41 @@ ENTRYPOINT [ "/sbin/tini", "--"]
 # Set Working Directory Context
 WORKDIR "/telescope"
 
-# Copy shared context
+# Copy package.jsons for each service
 COPY package.json .
+COPY ./tools/autodeployment/package.json ./tools/autodeployment/package.json
+COPY ./src/frontend/package.json ./src/frontend/package.json
 
 # -------------------------------------
 # Context: Dependencies
 FROM build AS dependencies
 
-# Get Entire Codebase
+# Install Production Modules! 
+# Disable postinstall hook in this case since we are being explict with installs
+RUN npm install --only=production --no-package-lock --ignore-scripts
+
+# Install Frontend Modules!
+RUN cd ./src/frontend && npm install --no-package-lock
+
+# Install Deployment Microservice Modules!
+RUN cd /telescope/tools/autodeployment && npm install --no-package-lock
```

Comment:
Same question here...

Reply:
Adding the same `--ignore-scripts` to ensure that no other hooks are picked up, one item to note is that this translates to using the default npm scripts and all other ones fail. So, as long as we are doing straight `install`, `build`, and `start` with the front-end not having such since we need the custom build command) it should not cause any trouble.